pkg/csi/service/service.go

```diff
@@ -106,13 +110,22 @@ func (s *service) BeforeServe(
 		log.Infof("configured: %q with clusterFlavor: %q and mode: %q", csitypes.Name, clusterFlavor, s.mode)
 	}()
 
+	var (
+		err error
+		cfg *cnsconfig.Config
+	)
+
+	// Initialize CO utility in Controller and Nodes
+	commonco.ContainerOrchestratorUtility, err = commonco.GetContainerOrchestratorInterface(ctx, common.Kubernetes, clusterFlavor, COInitParams)
```

Comment:
Can you execute this for controller only, as node does not require to use feature switch for 2.1 release.
Otherwise, this will also require us to update RBAC rules for nodes.

Reply:
Done